### PR TITLE
Feature/custom tag atttr

### PR DESCRIPTION
**Pull request by cossssmin**
This PR adds options for using:

- custom tag names 42aeb7efbbe4165cdf29402eb4850bf253c8b483
- custom attribute names 3b6d316899134d6cd8ac93f3e3c8cefab6881d49

README has been updated to reflect these changes.

---

I also renamed the license file, since it doesn't contain any markdown actualy, and not specifying an extension helps some editor themes give it the right icon:

image

Pictured above is the Material theme for VS Code.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,24 +10,13 @@
   <p>Import and process HTML Modules with PostHTML</p>
 </div>
 
-<h2 align="center">Install</h2>
+## Install
 
 ```bash
-npm i -D posthtml-modules
+npm i posthtml-modules
```
Comment (anikethsaha) [inline README.md:16]: can we add the `-D` as it should be installed as dev dep right !

Comment (cossssmin) [reply README.md]: Sure thing! Added back, sorry about that!

**Comment by Scrum** [comment]
@cossssmin publish v0.6.0

**Comment by cossssmin** [comment]
> @cossssmin publish v0.6.0

Thank you! :)

**Pull request merged**